CHANGELOG.md

```
@@ -3,9 +3,9 @@
 ## 1.13.0 (2025-12-29)
 
 ## What's Changed
-* feat: Add concurrent execution support with handle/session maps and inline queue by @shabaraba in https://github.com/shabaraba/vibing.nvim/pull/239
-* fix: Display plugin slash commands with plugin name in VibingSlashCommands by @shabaraba in https://github.com/shabaraba/vibing.nvim/pull/241
 
+- feat: Add concurrent execution support with handle/session maps and inline queue by @shabaraba in https://github.com/shabaraba/vibing.nvim/pull/239
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)